### PR TITLE
Add pld2sobp

### LINKED ISSUE #95: add pld2sobp converter

### PR DESCRIPTION
**Pull request by nbassler**
note there is also rst2sobp,  but that is located in PyTRiP since .rst is a TRiP file.
fixes #95

### CODE REVIEW

File: pymchelper/utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#
```
Comment (grzanka) [inline pymchelper/utils/__init__.py]: Does it make sense to add this to empty file ?

File: pymchelper/utils/pld2sobp.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+        self.x = [0.0] * self.spots
+        self.y = [0.0] * self.spots
+        self.w = [0.0] * self.spots       # MU weight
+        self.rf = [0.0] * self.spots      # fluence weight
+
+        j = 0
+
+        for i in range(len(elements)):
+            token = elements[i].split(",")
```
Comment (grzanka) [inline pymchelper/utils/pld2sobp.py]: What about following code ?
```
for element in elements:
   token = element.split(',')
```

File: pymchelper/utils/pld2sobp.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    parser.add_argument('fout', nargs='?', metavar="output_file.dat", type=argparse.FileType('w'),
+                        help="path to the SHIELD-HIT12A/FLUKA output file, or print to stdout if not given.",
+                        default=sys.stdout)
+    parser.add_argument("-v", "--verbosity", action='count', help="increase output verbosity", default=0)
+    parser.add_argument("-f", "--flip", action='store_true', help="flip XY axis", dest="flip", default=False)
+    parser.add_argument("-d", "--diag", action='store_true', help="prints additional diagnostics",
+                        dest="diag", default=False)
+    parser.add_argument("-s", "--scale", type=float, dest='scale',
+                        help="number of particles per MU.", default=_particles_per_mu)
+    #  parser.add_argument('-V', '--version', action='version', version=self.__version__)
```
Comment (grzanka) [inline pymchelper/utils/pld2sobp.py]: Can be replaced with:
```parser.add_argument('-V', '--version', action='version', version=pymchelper.__version__)```
and versioneer import at beginning

File: pymchelper/utils/pld2sobp.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+    args = parser.parse_args(args)
+
+    if args.verbosity == 1:
+        logging.basicConfig(level=logging.INFO)
+    if args.verbosity > 1:
+        logging.basicConfig(level=logging.DEBUG)
+
+    a = PLDRead(args.fin)
+    args.fin.close()
```
Comment (grzanka) [inline pymchelper/utils/pld2sobp.py]: Can't it be done using `with` construct ?

File: pymchelper/utils/__init__.py (hunk 1 above)
Comment (nbassler) [inline pymchelper/utils/__init__.py]: Yes, we need strong copyright on it. Out emptiness may not be copied nor reproduced without our authorization..

**Comment by nbassler** [comment]
@grzanka was the optional repaint column fixed?

**Comment by grzanka** [comment]
What do you mean by "optional repaint column" ?

**Comment by nbassler** [comment]
for the record: pld files sometimes have a 5th column for the layer descriptor, specifying the number of repaints for this layer.

**Comment by grzanka** [comment]
@nbassler solved the issue with repaints number column in commit e0f6e9e

**Pull request merged**